src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int MULTIPLIER_CAPACITY_AND_THRESHOLD = 2;
+    private int threshold;
+    private int capacity;
+    private int size;
+    private Node<K, V>[] nodes;
+
+    public MyHashMap() {
+        capacity = DEFAULT_CAPACITY;
+        threshold = (int) (capacity * LOAD_FACTOR);
+        nodes = new Node[capacity];
+    }
+
+    private static class Node<K, V> {
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            increaseArray();
+        }
+        Node<K, V> putNode = new Node<>(key, value, null);
+        int position = hash(key);
+        Node<K, V> positionNode = nodes[position];
+        if (positionNode == null) {
+            nodes[position] = putNode;
+            size++;
+            return;
+        }
+        Node<K, V> tempNode = positionNode;
+        while (positionNode != null) {
+            if ((key == positionNode.key) || (key != null && key.equals(positionNode.key))) {
```

Comment:
you use `Objects.equals()` on line 62. let's be consistent

Reply:
I redit it.